datafusion/sql/src/planner.rs

```diff
@@ -4849,6 +4864,24 @@ mod tests {
         quick_test(sql, expected);
     }
 
+    #[test]
+    fn test_double_quoted_literal_string() {
+        // Assert double quoted literal string is parsed correctly like single quoted one in specific dialect.
+        let dialect = &MySqlDialect {};
```

Comment:
Thanks for adding this test. Could you also add one showing that behavior hasn't changed when using the default dialect?

Reply:
> Thanks for adding this test. Could you also add one showing that behavior hasn't changed when using the default dialect?

Ok, I add such test soon.